util/constants.go

```diff
@@ -0,0 +1,5 @@
+package util
+
+const (
+	UserAgent = "azure/acr/tasks"
```

Comment:
can you move this to the tokenutil package or just a const in the token_util file, since it doesn't make sense as a part of generic util; add copyright headers to the file if you decide on the other file